easybuild/easyconfigs/h/HBase/HBase-1.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'HBase'
+version = '1.0.2'
+
+homepage = 'http://hbase.apache.org/'
+description = """Apache HBase. is the Hadoop database, a distributed, scalable, big data store. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin.tar.gz']
```

Comment:
well, it would be sort of clear from the toolchain being used

if it's basically Java code, then it's good as is

I assume the `bin/hbase` is then a bash script or somesuch?